drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/panel/AnalysisReportScreenViewImpl.java

```diff
@@ -107,11 +107,27 @@ public void clearIssue() {
     }
 
     @Override
+    public void showStatusComplete() {
```

Comment:
Tried to look for better images from Bootstrap, but looks like PatternFly alerts are just what I wanted. Thank you for the tip.